README.md

```diff
@@ -112,3 +112,5 @@ http://www.apache.org/licenses/LICENSE-2.0
 
 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 
+## Kubernetes Support
+Check out the [dashboards/k8s-build](https://github.com/datastax/metric-collector-for-apache-cassandra/tree/master/dashboards/k8s-build) directory for a guide on using this project along with Kubernetes.
```

Comment:
This can be a relative link `dashboards/k8s-build`

Reply:
Fixed, I wasn't sure how it would behave if you were on the README.md page in GH, not just the repository index.